### PR TITLE
WIP: Improve UI

### PR DESCRIPTION
**Pull request by christeredvartsen**
The UI should be intuitive, and contain docs for the whole process.

### CODE REVIEW

**Comment by christeredvartsen** [comment]
Must be fixed:

- [ ] Submit button handler for registering config to S3
- [ ] Fetch repository config when selecting repository so we can check the teams currently set up

File: hookd/templates/form.html [hunk 1]
```diff
@@ -1,37 +1,168 @@
...
+        );
+
+        var select = document.getElementById("repository-select")
+        var repository = select.options[select.selectedIndex].value;
+
+        if (!repository) {
+            return;
+        }
+
+        var teamsSpinnerText = document.createElement("p");
```
Comment (ambientsound) [inline hookd/templates/form.html]: All good so far. But here it starts to get a bit crazy when you programatically create DOM elements. Isn't there another way?

Comment (christeredvartsen) [reply hookd/templates/form.html]: Well, we could have them in the DOM and move them around with JS instead and swap visibility along the way. Or we could use a lib that makes creating elements a lot easier than with vanilla JS.

Comment (christeredvartsen) [reply hookd/templates/form.html]: I can add a commit that instead of creating elements using JS just moves existing elements around.

**Pull request merged**